samples/ControlCatalog.iOS/Info.plist

```diff
@@ -16,7 +16,6 @@
 	<array>
 		<integer>1</integer>
 		<integer>2</integer>
-        <integer>3</integer>
```

Comment:
I removed this since DeviceType 3 is used for tvOS, which breaks deployment for iOS.